application/tests/cre_main_test.py

```diff
@@ -378,7 +378,9 @@ def test_parse_standards_from_spreadsheeet(self) -> None:
                 "Standard CWE (from ASVS)-hyperlink": "https://example.com/cwe19876",
                 "Link to other CRE": "FooBar",
                 "Standard NIST 800-53 v5": "SA-22 Unsupported System Components",
-                "Standard NIST 800-53 v5-hyperlink": "https://example.com/nist-800-53-v5",
```

Comment:
is this change necessary?
it's just adding parentheses


same for all changes in this file